.github/workflows/rosa-multi-az-cluster-clear.yml

```diff
@@ -0,0 +1,66 @@
+name: Multi-AZ Clusters - Clear
```

Comment:
@andyuk1986 my suggestion would be to name this workflow to be `rosa-multi-az-cluster-undeploy.yml` and the name to be `Multi-AZ Clusters - Undeploy` that would make the workflow consistent with the downstream tasks such as `task undeploy. Its a great Idea by the way to have the steps sorted this way, thanks for doing that :)

Reply:
thanks a lot for the suggestion, I have renamed the job and the file correspondingly.